endpoint.cpp

```diff
@@ -371,14 +371,28 @@ int UartEndpoint::open(const char *path, speed_t baudrate)
 
     tc.c_iflag &= ~(IGNBRK | BRKINT | ICRNL | INLCR | PARMRK | INPCK | ISTRIP | IXON);
     tc.c_oflag &= ~(OCRNL | ONLCR | ONLRET | ONOCR | OFILL | OPOST);
-    tc.c_lflag &= ~(ECHO | ECHOE | ECHOK | ECHOCTL | ECHOKE | ECHONL | ICANON | IEXTEN | ISIG | TOSTOP);
-    tc.c_cflag &= ~(CSIZE | PARENB | CBAUD | CRTSCTS);
+
+    tc.c_lflag &= ~(ECHO | ECHOE | ECHOK | ECHOCTL | ECHOKE | ECHONL | ICANON | IEXTEN | ISIG);
+
+    /* never send SIGTTOU*/
+    tc.c_lflag &= ~(TOSTOP);
+
+    /* disable flow control */
```

Comment:
Well - on PX4 mavlink module, there's this line https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_main.cpp#L824 that actually tries to enable flow control - couldn't this create some problems if we always disable it?